simulator/EndCondition.ts

```diff
@@ -61,7 +66,7 @@ export class EndCondition implements IEndCondition {
                 return true;
             }
         } else if (liveWarriors.length === 1) {
```

Comment:
I wonder if for multi-warrior rounds (>2) we could have an event for warrior death. Certainly not a big thing, maybe a nice to have in like 10 years time